setUpApplicationDefaults.js

```diff
@@ -64,7 +69,25 @@ export default async (app) => {
     // Attach always useful middleware
     app.use(helmet());
     app.use(cors());
-    app.use(prometheus());
+
+    // Add prometheus monitoring metrics
+    if (config.prometheusMetrics) {
+        const metricsConfig = { ...config.prometheusMetrics };
+        if (config.prometheusMetrics.port) {
+            app.metricsApp = express();
+        }
+        app.use(promBundle({
+          metricsApp: app.metricsApp, // its fine to pass an undefined here, if we don't want an alternative app for metrics
+          autoregister: !Boolean(config.prometheusMetrics.port),  // if no port defined, we register on primary express app
+          includeMethod: true,
+          includePath: true,
+          promClient: {
+            collectDefaultMetrics: { },
+            ...config.prometheusMetrics.promClient, // see https://github.com/siimon/prom-client for config options
+          },
+        }));
+    }
+    // TODO we need to terminate app.metricsApp too somehow on shutdown
```

Comment:
This is now handled, comment should be removed!

Reply:
Done